backend/app/schemas/metrics.py

```diff
@@ -41,7 +50,12 @@ class PipelineMetrics(BaseModel):
             "example": {
                 "time_window": {"start": "2025-08-25T10:00:00Z", "end": "2025-08-25T10:01:00Z", "time_window": 60},
                 "inference": {
-                    "latency": {"avg_ms": 15.1, "min_ms": 12.3, "max_ms": 30.4, "p95_ms": 25.4, "latest_ms": 15.6}
+                    "latency": {"avg_ms": 15.1, "min_ms": 12.3, "max_ms": 30.4, "p95_ms": 25.4, "latest_ms": 15.6},
+                    "throughput": {
+                        "avg_inferences_per_second": 66.7,
+                        "total_inferences": 4000,
+                        "peak_inferences_per_second": 85.2,
```

Comment:
```suggestion
                        "avg_requests_per_second": 66.7,
                        "total_requests": 4000,
                        "max_requests_per_second": 85.2,
```
Small renaming suggestions. "requests" instead of "inferences" (which I think is a contraption of inference requests), and "max" instead of "peak" for consistency with the latency attributes ("max_ms").